docs/man/git-lfs-migrate.1.ronn

```diff
@@ -38,6 +38,11 @@ git-lfs-migrate(1) - Migrate history to or from git-lfs
 * `--everything`:
     See [INCLUDE AND EXCLUDE (REFS)].
 
+    Note: Git refs are case sensitive on all platforms in packed from (see
+    `git-pack-refs(1)). On case insensitive file systems, e.g. NTFS on Windows
```

Comment:
I think this should be “packed form” and “case-insensitive” (with a hyphen).  Also, it looks like the backtick is missing after `git-pack-refs(1)`.

Reply:
Fixed. Thank you 👍